### ISSUE TITLE
Compare to mkcert

### ISSUE THREAD

**Issue by vt-alt** [txthinking/mad#4]
What do you think about this https://github.com/FiloSottile/mkcert

**Comment by txthinking** [comment]
yeah, i know this, looks like it does not support create ca same as domain cert on command line based on the readme.
Or the concepts of concern are different, mad, there are only two concepts root ca and cert.
